dist/style.css

```diff
@@ -0,0 +1,171 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Lobster+Two&family=Roboto:wght@1,300&display=swap');
+
+*{
+    font-family:  'Lobster Two', cursive, 'Roboto', sans-serif;
+    padding: 0;
+    margin: 0;
+    /* color: white; */
```

Comment:
Please remove this commented code and other similar commented code in this file and others.